dmtestutils/api_model_stubs/brief.py

```diff
@@ -65,10 +65,14 @@ def __init__(self, **kwargs):
             and ("framework_name" not in kwargs and "frameworkName" not in kwargs)
             and ("framework_family" not in kwargs and "frameworkFramework" not in kwargs)
         ):
-            self.response_data.update(self._format_framework(self.response_data["frameworkSlug"], oldstyle=True))
+            # The brief model serialisation includes both new-style and old-style framework keys,
+            # and the deprecated `frameworkFramework` field. Consumers should use `brief['framework']['family']`,
+            # not `brief['frameworkFamily']` or `brief['frameworkFramework]`, but we should still make sure
+            # our test stubs reflect the model.
+            self.response_data.update(
+                self._format_framework(self.response_data["frameworkSlug"], newstyle=True, oldstyle=True)
```

Comment:
tiny nitpick but would `old_style` and `new_style` be more idiomatic python?

Reply:
I'm happy to change it, although I should note that underscores aren't necessarily required, PEP8 just says variable names should be lowercase

> with words separated by underscores as necessary to improve readability

https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Do you find it harder to read the names as is?

Reply:
It is not a big deal, intellij will flag it as a typo as-is though.